File: benchmark/fib.py
# Recursive calls and branch-heavy integer work.
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)


t0 = time.perf_counter()
result = fib(28)
print((time.perf_counter() - t0) * 1000)

File: benchmark/fib.lua
-- Recursive calls and branch-heavy integer work.
local function fib(n)
    if n <= 1 then
        return n
    end
    return fib(n - 1) + fib(n - 2)
end

local t0 = os.clock()
local result = fib(28)
print((os.clock() - t0) * 1000)
